scripts/setup.py

```diff
@@ -28,6 +40,28 @@ def setup(dataDir):
         filename = os.path.basename(file)
         path = os.path.join(dataDir, filename)
         open(path, "wb").write(response.content)
+    
+    # Loading the temp.zip and creating a zip object
+    with ZipFile("../data/vosk-model-small-en-us-0.15.zip", 'r') as zObject:
+    
+        # Extracting the zip
+        zObject.extractall(path="../data")
+    
+    # Delete the zip file
+    os.remove("../data/vosk-model-small-en-us-0.15.zip")
```

Comment:
I think it would be better to generalize this and move it inside the `for` above it.
Check if the file extension is `.zip`, in which case you extract it and delete the `.zip` file.